plugins/woocommerce/src/Blocks/BlockTypes/MiniCartFooterBlock.php

```diff
@@ -92,6 +102,67 @@ protected function get_cart_instance() {
 		return null;
 	}
 
+	/**
+	 * Computes the total items description text based on which settings are enabled.
+	 *
+	 * @return string The description text for the total items, or empty string if none are enabled.
+	 */
+	private function get_total_items_description() {
```

Comment:
Similar as above with regards to the function name:


```suggestion
	private function get_totals_item_description() {
```